src/cli/repo/repo.go

```diff
@@ -84,3 +103,65 @@ func connectCmd() *cobra.Command {
 func handleConnectCmd(cmd *cobra.Command, args []string) error {
 	return cmd.Help()
 }
+
+func maintenanceCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   maintenanceCommand,
+		Short: "Run maintenance on an existing repository",
+		Long:  `Run maintenance on an existing repository to keep it running smoothly`,
+		RunE:  handleMaintenanceCmd,
+		Args:  cobra.NoArgs,
+	}
+}
+
+func handleMaintenanceCmd(cmd *cobra.Command, args []string) error {
+	ctx := cmd.Context()
+
+	s := control.Safety(utils.MaintenanceSafetyFV)
+	if err := validateMaintenanceFlags(s); err != nil {
+		return err
+	}
+
+	r, _, err := utils.GetAccountAndConnect(ctx)
+	if err != nil {
+		return print.Only(ctx, err)
+	}
+
+	defer utils.CloseRepo(ctx, r)
+
+	m, err := r.NewMaintenance(
+		ctx,
+		s,
+		utils.QuickMaintenanceFV,
+		utils.ForceMaintenanceFV)
+	if err != nil {
+		return print.Only(ctx, err)
+	}
+
+	err = m.Run(ctx)
+	if err != nil {
+		return print.Only(ctx, err)
+	}
+
+	return nil
+}
+
+func validateMaintenanceFlags(s control.Safety) error {
+	_, ok := control.SafetyValues[s]
+	if !ok {
+		values := maps.Keys(control.SafetyValues)
```

Comment:
`values = keys` hits the ocd part of me hard.
```suggestion
		safetys := maps.Keys(control.SafetyValues)
```